packit/api.py

```diff
@@ -1285,7 +1287,7 @@ def create_srpm(
         output_file: Optional[str] = None,
         upstream_ref: Optional[str] = None,
         srpm_dir: Union[Path, str] = None,
-        bump_version: bool = True,
+        bump_version: Optional[bool] = None,
```

Comment:
we should deprecate and rename this to `bump_release` in future

Reply:
https://github.com/packit/packit-service/issues/1760#issuecomment-1328669054 suggests naming the option just `bump`, but I've thought it might be a bit more descriptive, so I initially named it `bump_version_release` and then shorten it to just `bump_version` because of the arguments of the existing methods.

Do you think I should rather rename all to `bump_release`? Or any other suggestions, @packit/the-packit-team ?

Reply:
I suggested `bump` to be consistent with the CLI name.

But I don't really like it as I wrote here: https://github.com/packit/packit.dev/issues/570.

My English is really bad, but as far as I understand this word and our code, for me we are not *bumping* the release in the specfile,  we are *updating* it with different templates chosen by the `release_suffix` value.

But as I said my english is bad so if you think that `bump_release` is not misleading then I agree with it :)

Reply:
I agree with Maja, I think *to bump* (version/release) is generally understood as *to increment* (https://stackoverflow.com/a/4181188), and it would make more sense to call this *update*.

Reply:
+1 for something like `update_release`, if that better describes the behaviour.

Reply:
+ 1 for `update` rather than `bump`

Reply:
+1 update